### PR TITLE
Moj wspanialfesg com

### PR DESCRIPTION
**Pull request by Banndzior**
<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

**Comment by LosowyNick** [comment]
great!

File: kamil-mijacz/empty.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
```
Comment (LosowyNick) [inline kamil-mijacz/empty.ts:3]: a co to?

File: kamil-mijacz/empty.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ *
```
Comment (Banndzior) [inline kamil-mijacz/empty.ts:4]: to glupie

**Review by LosowyNick — APPROVED** [review]

**Review by AnetaSyed — APPROVED** [review]

**Review by justynalem — APPROVED** [review]

**Review by anoGitHub — APPROVED** [review]

**Review by Wlodirstmn — APPROVED** [review]

**Review by MaciejPanasiuk — APPROVED** [review]
Srogi kod

**Review by SebastianTyszko — APPROVED** [review]
superb

**Review by LosowyNick — CHANGES_REQUESTED** [review]
:(

**Review by Wlodirstmn — COMMENTED** [review]
sprawdz kod

**Review by LosowyNick — APPROVED** [review]

**Pull request merged**